ui/app/templates/variables/variable/index.hbs

```diff
@@ -72,59 +81,65 @@
   <div class="boxed-section">
     <div class="boxed-section-head">
       Key/Value Data
-      <CopyButton
-        class="pull-right"
-        @compact={{true}}
-        @border={{true}}
-        @clipboardText={{stringify-object this.model.items}}
-          />
     </div>
     <div class="boxed-section-body is-full-bleed">
       <JsonViewer @json={{this.model.items}} />
     </div>
   </div>
 {{else}}
-  <ListTable class="variable-items" @source={{this.sortedKeyValues}} @sortProperty={{this.sortProperty}} @sortDescending={{this.sortDescending}} as |t|>
-    <t.head>
-      <t.sort-by @prop="key">Key</t.sort-by>
-      <t.sort-by @prop="value">Value</t.sort-by>
-    </t.head>
-    <t.body as |row|>
-      <tr data-test-var={{row.model.key}}>
-        <td>
-          {{row.model.key}}
-        </td>
-        <td colspan="3" class="value-cell">
+  <Hds::Table class="variable-items"
+  @model={{this.sortedKeyValues}}
+  @sortBy={{this.sortProperty}}
+  @sortOrder={{if this.sortDescending "desc" "asc"}}
+  @columns={{array
+    (hash
+      key="key"
+      label="Key"
+      isSortable=true
+      width="200px"
+    )
+    (hash
+      key="value"
+      label="Value"
+      isSortable=true
+    )
+  }}
```

Comment:
An astute observe may note that some of the `<Hds::Table>`s here have a `@columns` property, where others do the `<:head as |H|>` and `H.Tr` thing. Helios tables allow us to do either, and the best argument for doing the former has to do with sorting being baked in.